### ISSUE TITLE
Wrist arthritis

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10714]
**Preferred term label:**
Wrist arthritis

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/browse/term/HP:0003019

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Wrist arthritis
Scaphoid nonunion advanced collapse (SNAC)
Scapholunate advanced collapse (SLAC)

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001197
- label: Wrist arthritis
- parents: [Arthritis (HP:0001369)](https://hpo.jax.org/app/browse/term/HP:0001369)
- pmid: PMID:30285392
- def: Inflammation of the wrist joint.
- comment: Symptoms of arthritis include pain, swelling, redness, stiffness, and loss of motion of the involved joint. Three primary types of arthritis affect the human wrist: osteoarthritis, rheumatoid arthritis, and post-traumatic arthritis.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10714
- synonyms:
